src/Scripts/DockerLocalBuild.ps1

```diff
@@ -12,23 +12,37 @@
 #       "musl" CLib for the Alpine 3.7 sample. More generally, use "gnu" build output for glibc-
 #       based distros and "musl" build output for musl-libc-based distros.
 
+<#
```

Comment:
Yay PowerShell comment blocks! Would you mind also moving the above Usage comment to this? probably in an `.EXAMPLE` section

Reply:
I don't need to add the parameters to the documentation comment because the parameters are documented inline.